opencti-worker/src/worker.py

```diff
@@ -369,7 +370,13 @@ def data_handler(  # pylint: disable=too-many-statements, too-many-locals
                     )
                     # Add expectations to the work
                     if work_id is not None:
+                        # add expectation with worker user and not impersonated user
```

Comment:
Instead of doing that maybe we can:
- Change the name of the current self.api to self.impersonate_api
- Add a new self.api based on new OpenCTIApiClient with perform_health_check = false
- Use self.impersonate_api for everything except self.api for add_expectations